dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_homepage_v2.feature

```diff
@@ -6,7 +6,8 @@ Feature: Using the teacher homepage
     Given I create a teacher named "Teacher Hank"
 
     # This line can be removed when the new teacher homepage is the default
-    Then I use a cookie to mock the DCDO key "teacher-homepage-v2" as "true"
+    Then I use a cookie to mock the DCDO key "teacher-local-nav-v2" as "true"
+    And I use a cookie to mock the DCDO key "teacher-homepage-v2" as "true"
```

Comment:
can you please update the comment to match the code change? Also I am curious how I would know if "teacher homepage is the default"